src/panel.js

```diff
@@ -472,13 +472,10 @@ const addHeaderWithBackArrow = (fragment) => {
 };
 
 
-// Add additional domains that need to be shown in the "Sites Allowed" panel here
```

Comment:
This feels like a artifact from a bad `rebase`. These file changes should be reverted.

Reply:
Added back the changes from 13b9a16.